extensions/amp-gfycat/amp-gfycat.md

```diff
@@ -70,7 +76,7 @@ Gfycat allows many aspect ratios.
 
 To specify the width and height in the code, copy it from the embed URL:
 
-1. Go to https://gfycat.com/name, where name is the Gfycat ID.
+1. Go to [https://gfycat.com/](https://gfycat.com/)`name`, where name is the Gfycat ID.
```

Comment:
From looking at https://travis-ci.org/ampproject/amphtml/jobs/490580706#L810, this is being treated as a single link. Perhaps you can put the entire string in backticks, like this?
```
Go to `https://gfycat.com/name`, where name is the Gfycat ID.
```

Reply:
Saw your comment to late, I went with the following. What do you think? 

```md
[https://gfycat.com/](https://gfycat.com/)`name`
```

Reply:
My original suggestion was based on the fact that this is meant to be an example, and not an actual clickable link. Either way, if the link checker is satisfied, I have no objections :)